cmake/noa/library.cmake

```diff
@@ -32,7 +32,11 @@ function(noa_library)
   endif()
 
   foreach(private_header IN LISTS NOA_LIBRARY_PRIVATE_HEADERS)
-    list(APPEND ABSOLUTE_PRIVATE_HEADERS "${INCLUDE_PREFIX}/${NOA_LIBRARY_NAME}_${private_header}")
+    if(NOA_LIBRARY_VARIANT)
+        list(APPEND ABSOLUTE_PRIVATE_HEADERS "../${INCLUDE_PREFIX}/${NOA_LIBRARY_NAME}_${private_header}")
```

Comment:
Can we indent using 2 spaces? Looks like these two `list` lines are using 4

Reply:
yeap